### ISSUE TITLE
Scikit-learn import

### ISSUE THREAD

**Issue by DannyBaetler** [holzschu/Carnets#113]
Very nice Jupyter implementation for IOS! 
I am not able to install Scikit-learn on Carnets. Can you implement it or could you give me a tip how to install it?

Thank you.

**Comment by alimmemon** [comment]
I am having the same issue and would like to like install sklearn as well.

**Comment by holzschu** [comment]
Hi, 
sklearn requires scipy and a C compiler. It is not possible to install it on Carnets (yet).

**Comment by ifuchs** [comment]
I see that scipy is part of the new beta but am I correct that scikit-learn is still not installable?

**Comment by holzschu** [comment]
That is correct. scipy is a requirement for installing scikit-learn, but it's not sufficient. We still need to compile some C modules for scikit-learn. But first I need to test scipy intensely, then release it.

**Comment by holzschu** [comment]
The version with scikit-learn is now available for TestFlight: https://testflight.apple.com/join/UbNIjkjp
(it is a separate app because it is substantially larger with scipy and sklearn).
